gui/build.py

```diff
@@ -79,6 +80,20 @@ def do_bundle():
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
+    # create a intaller
+    env = dict(os.environ)
+    p = re.search('D:\\\\Python27\\\\Lib', env['PYTHONPATH'])
+    python_path = p.group(0)
+    gui_path = os.path.realpath('.')
+    moz_path = os.path.realpath('..')
+    include_path = '--include-path=%s;%s;%s//site-packages;%s ' % (gui_path, moz_path, python_path, python_path)
+    icon_arg = '--icon=wininst/app_icon.ico '
+    target_arg = '--target-name=mozregression-gui.exe --target-dir=dist'
+    base_name = '--base-name=Win32GUI '
+    arg = icon_arg + base_name + include_path + target_arg
+    call(sys.executable, py_script('cxfreeze'), arg, 'mozregui/main.py')
```

Comment:
call() require you to list the arguments one by one - that is:
call('cd', 'mozregui')
but you can put them in a list and then use * (http://stackoverflow.com/questions/3394835/args-and-kwargs)
args = ['cd', 'mozregui']
call(*args)

So here you should build args this way for example:
args = []
args.append('--icon=wininst/app_icon.ico')  # note: no leading space
args.append('--base-name=Win32GUI')
...
args.append('mozregui/main.py')
call(sys.executable, py_script('cxfreeze'), *args)